spec-main/api-app-spec.ts

```diff
@@ -674,25 +674,30 @@ describe('app module', () => {
 
   describe('getPath("logs")', () => {
     const logsPaths = {
-      'darwin': path.resolve(homedir(), 'Library', 'Logs'),
-      'linux': path.resolve(homedir(), 'AppData', app.name),
-      'win32': path.resolve(homedir(), 'AppData', app.name),
+      'darwin': path.resolve(homedir(), 'Library', 'Logs', 'Electron'),
+      'linux': path.resolve(homedir(), 'AppData', 'Electron', 'logs'),
```

Comment:
The second component in this call should be '.config' rather than 'AppData'. This would cause the 'has no logs directory by default' test to incorrectly pass if  ~/.config/Electron/logs existed

That bug looks like a copy-paste error that predates this PR

Reply:
fixed!